### ISSUE TITLE
Weird parsing of square brackets content

### ISSUE THREAD

**Issue by L3nn0x** [Yamakaky/dcpu#14]
When trying to use square brackets, the parsing inside is weirdly done.

Minimal example : `set [test + J + 1], 1`

Expected result : compiles and actually precomputes test + 1

Actual result : doesn't compile and I get the error `Error: Error { kind: UnknownLabel("J"), state: State { next_error: None, backtrace: None } }`.

**Comment by Yamakaky** [comment]
Yeah, I know, brackets are not well parsed. https://github.com/Yamakaky/dcpu/blob/master/src/assembler/parser/nom_parser.rs#L131-L144
